data/pages-content/sponsor/information.md

```diff
@@ -242,26 +237,26 @@ at all times.
 # Sponsored Content
 
 <details>
-  <summary><h2>Talks | Workshops/Tutorials | Posters</h2></summary>
+  <summary><h3>Talks | Workshops/Tutorials | Posters</h3></summary>
```

Comment:
To be honest, it took me some time to understand that I have to click the text to see more information. And my mind is still wondering why for example by clicking the link here: https://github.com/EuroPython/website/blob/58cf56f6c5fdf9edf26898161dcfffcb24e3633e/data/pages-content/sponsor/information.md?plain=1#L11-L12 I got redirected to something with different name: "Talks | Workshops/Tutorials | Posters"

Maybe we should not hide the content? But if we do, I would probably try to say that you need to click explicitly without 👇 emoji 🤓

Reply:
Or can we open the text automatically right after the user click the link on top?

Reply:
> Or can we open the text automatically right after the user click the link on top?

That would be really nice. I am not sure how easy it is. Not familiar with the components. @patrick91 any thoughts?